pettingzoo/utils/wrappers/multi_episode_env.py

```diff
@@ -13,21 +13,30 @@ class MultiEpisodeEnv(BaseWrapper):
     When there are no more valid agents in the underlying environment, the environment is automatically reset.
     After `num_episodes` have been run internally, the environment terminates normally.
     The result of this wrapper is that the environment is no longer Markovian around the environment reset.
+
+    When `starting_utility` is used, all agents start with a base amount of health points (think of this as poker chips).
+    Whenever the agent gets a negative reward, this value is subtracted from starting utility.
+    Whenever the agent gets a positive reward, it is added to the starting utility.
+    Agents which run out of starting utility are terminated.
     """
 
-    def __init__(self, env: AECEnv, num_episodes: int):
+    def __init__(
+        self, env: AECEnv, num_episodes: int, starting_utility: float | None = None
+    ):
         """__init__.
 
         Args:
             env (AECEnv): env
             num_episodes (int): num_episodes
+            starting_utility (float | None): starting_utility
```

Comment:
It's odd but in these types of Args: parts I think type hinting in our other classes usually does `Optional[float]` rather than `| None`

Reply:
Fixed.